packages/lit-dev-content/samples/tutorials/custom-attribute-converter/04/before/date-converter.ts

```diff
@@ -0,0 +1,11 @@
+import {ComplexAttributeConverter} from 'lit';
```

Comment:
Do we encourage type-only imports?
```suggestion
import type {ComplexAttributeConverter} from 'lit';
```